karton/core/main.py

```diff
@@ -168,6 +179,9 @@ def main() -> None:
 
     subparsers.add_parser("list", help="List active karton binds")
 
+    logs_parser = subparsers.add_parser("logs", help="Start streaming logs")
+    logs_parser.add_argument("--filter", help="Service identity filter", required=False)
+
```

Comment:
AFAIK, we can also specify the level of logs to listen for: https://github.com/CERT-Polska/karton/blob/43ce3ac1f93d56f3bc088c3fb7c34323c2be8150/karton/core/karton.py#L334

Maybe it'd be also worth putting here?

Reply:
Well, I thought about this but doing this will require either:
* listening on a single log level (not really user friendly)
* or listening on all log levels and filtering them by hand in `CliLogger`

IMO it's not critical and can be added in the future

Reply:
> IMO it's not critical and can be added in the future

Makese sense.

> Well, I thought about this but doing this will require either:

Why though? Can't we just accept `info/log/error` etc, and then use `*` as the default?

Reply:
We could but this would mean that we're listening only on a single log level.